gdp/small_batch/gdp_small_batch.py

```diff
@@ -9,99 +17,200 @@
 
 
 def build_small_batch():
+    """
+    The function build the GDP model for the small batch problem.
+
+    Args:
+        None
+    
+    Returns:
+        m: pyomo.ConcreteModel
+    """
     NK = 3
 
     # Model
     m = pe.ConcreteModel()
 
     # Sets
-    m.i = pe.Set(initialize=['a', 'b'])  # Set of products
+    m.i = pe.Set(initialize=['a', 'b'], doc='Set of produncts')  # Set of products, i=1,2
     m.j = pe.Set(initialize=['mixer', 'reactor',
-                             'centrifuge'])  # Set of stages
+                             'centrifuge'])  # Set of stages, j=1,2,3
```

Comment:
Same as above

Reply:
Switched it. As well as the k on the bottom